Firestore/core/src/firebase/firestore/local/local_documents_view.mm

```diff
@@ -215,6 +217,38 @@
   return results;
 }
 
+// It is possible that a `PatchMutation` can make a document match a query, even
```

Comment:
Optional: I'm not against adding a new comment for this function in the header, but you could simply move this comment to the header (replacing the current one) as well.

Reply:
Done.